### ISSUE TITLE
Dropdown : The luminosity ratio is less than 4.5:1 for 'Disabled example with defaultSelectedKey' text.

### ISSUE THREAD

**Issue by 999-Dev** [OfficeDev/office-ui-fabric-react#10902]
**User Experience:**
Low vision user will face difficulty is luminosity ratio of text 'Disabled example with defaultSelectedKey' will be less then 4.5:1. 

**Test Environment:**
OS: Version: RS5_Prerelease(1903 OS Build 19005.1)
URL: https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown
Edge Browser Version:
Microsoft Edge: 44.19005.1.0
Microsoft EdgeHTML: 18.19005
Chrome Browser Version:
Chrome: 77.0.3865.120

**Usages:**
Basic Dropdowns

**Repro Steps:**
1.https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown in Chrome.
2.Navigate to 'Basic input' button and activate it.
3.Navigate to 'Dropdown' link and activate it.
4.'Dropdown' pane will be appeared.
5.Run K4W, and verify the issue.

**Actual Result:**
Luminosity ratio is less than 4.5:1, so the low vision user will face difficulty.

**Expected Result:**
Luminosity ratio should be greater than 4.5:1.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachment:**
![image](https://user-images.githubusercontent.com/55537167/67088660-5b0ce800-f1c3-11e9-9e99-f4b3d147cca7.png)
![image](https://user-images.githubusercontent.com/55537167/67088704-72e46c00-f1c3-11e9-93dd-a166adfa8c19.png)

**Comment by marygans** [comment]
Hi @999-Dev thanks for submitting this issue! 

I have verified this issue. It looks like disabled `Labels` in general are slotted to `disabledText` in `semanticColors`, and is the cause of this issue. I will talk to designers to see how we will proceed with this. 
Thanks!

**Comment by marygans** [comment]
Hi @999-Dev  according to accessibility guidelines:
"Text or images of text that are part of an inactive [user interface component](https://www.w3.org/TR/WCAG21/#dfn-user-interface-components), that are [pure decoration](https://www.w3.org/TR/WCAG21/#dfn-pure-decoration), that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement."

So I will conclude that this is not an issue according to the guidelines above. 

Thanks!
